libsolidity/ast/AST.cpp

```diff
@@ -981,26 +994,8 @@ TryCatchClause const* TryStatement::successClause() const
 	return m_clauses[0].get();
 }
 
-TryCatchClause const* TryStatement::panicClause() const
-{
-	for (size_t i = 1; i < m_clauses.size(); ++i)
-		if (m_clauses[i]->errorName() == "Panic")
-			return m_clauses[i].get();
-	return nullptr;
-}
+TryCatchClause const* TryStatement::panicClause() const { return findClause(m_clauses, "Panic"); }
```

Comment:
Since this is an implementation file, I'd wrap these properly:
```suggestion
TryCatchClause const* TryStatement::panicClause() const {
	return findClause(m_clauses, "Panic");
}
```